libcxx/include/complex

```diff
@@ -708,13 +708,14 @@ operator/(const complex<_Tp>& __z, const complex<_Tp>& __w)
     if (__libcpp_isfinite_or_builtin(__logbw))
     {
         __ilogbw = static_cast<int>(__logbw);
-        // ldexp is equivalent to scalbn when FLT_RADIX == 2
-        __c = ldexp(__c, -__ilogbw);
-        __d = ldexp(__d, -__ilogbw);
+        // Promote to double as scalbn is not generic
```

Comment:
@griwes I'm testing this on Windows now, but I want to get your opinion first. In lieu of implementing a function to correctly dispatch to `scalbnf/scalbnl/scalbn` I though this seemed fine.